src/index.tsx

```diff
@@ -210,11 +215,13 @@ class H5AudioPlayer extends Component<PlayerProps> {
   }
 
   handleClickRewind = (): void => {
-    this.setJumpTime(-this.props.progressJumpStep)
+    const jumpStep = this.props.progressJumpSteps.backward != undefined ? this.props.progressJumpSteps.backward : this.props.progressJumpStep
+    this.setJumpTime(-jumpStep)
   }
 
   handleClickForward = (): void => {
-    this.setJumpTime(this.props.progressJumpStep)
+    const jumpStep = this.props.progressJumpSteps.forward != undefined ? this.props.progressJumpSteps.forward : this.props.progressJumpStep
+    this.setJumpTime(jumpStep)
```

Comment:
This is against some eslint/prettier rules: `max-len`, `no-undef`.

Also, could you use destructuring instead of accessing props multiple times?
```js
const { progressJumpSteps, progressJumpStep } = this.props
const jumpStep = progressJumpSteps.backward || progressJumpStep
```

Reply:
Yeah absolutely, just fixed it. Thanks for the feedback.